mock/Cargo.toml

```diff
@@ -23,7 +23,7 @@ optional = true
 version = "0.0"
 
 [dependencies.rusoto_core]
-version = "0.25.0"
+version = "> 0.25.0"
```

Comment:
This was a last ditch effort to get the build passing: I don't think it's actually required.